src/adapters/zwave/zwave-classifier.js

```diff
@@ -44,16 +44,18 @@ class ZWaveClassifier {
         }
     }
 
-    initOnOffSwitch(node, valueId) {
-        node.type = THING_TYPE_ON_OFF_SWITCH;
+    initOnOffThing(node, valueId, type) {
+        node.type = type;
         node.properties.set('on',
                             new ZWaveProperty(node, 'on', 'boolean', valueId));
```

Comment:
I've made this change. It may be worth considering consistent naming with the CSS, graphics, Thing class etc. as well.